fixlib/fixlib/baseresources.py

```diff
@@ -1094,6 +1094,23 @@ class DatabaseInstanceStatus(Enum):
     UNKNOWN = "unknown"
 
 
+@define(eq=False, slots=False)
+class BaseDatabaseInstanceType(BaseType):
```

Comment:
As it is now, can we just use BaseInstanceType?

Reply:
We can do something like:
```python
@define(eq=False, slots=False)
class BaseDatabaseInstanceType(BaseInstanceType):
    kind: ClassVar[str] = "database_instance_type"
    kind_display: ClassVar[str] = "Database Instance Type"
    kind_description: ClassVar[str] = "A database instance type."
    metadata: ClassVar[Dict[str, Any]] = {"icon": "database", "group": "database"}
```
Here we change the metadata and `kind` properties, and other fields inherit from `BaseInstanceType`